docs/readmes/orc8r/how_to_deploy_using_juju.md

```diff
@@ -0,0 +1,103 @@
+---
+id: how_to_deploy_using_juju
+title: How-To: Deploy Orchestrator using Juju (Beta)
+hide_title: true
+---
+
+# How-To: Deploy Orchestrator using Juju (Beta)
+
+The goal of this document is to detail how to deploy Magma's Orchestrator on any Kubernetes
+cluster. To do so, we will set up a Kubernetes cluster, bootstrap a Juju controller, deploy Magma
+Orchestrator and configure A records for Orchestrator services to be accessible from anywhere.
+
+## Pre-requisites
+
+- Ubuntu 20.04 machine with internet access
+- A public domain
+
+## 1. Set up your management environment
+
+From a Ubuntu 20.04 machine, install the following tools:
+
+- [Juju](https://juju.is/docs/olm/installing-juju)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+
+## 2. Create a Kubernetes cluster and bootstrap a Juju controller
+
+Select a Kubernetes environment and follow the guide to create the cluster and bootstrap
+a Juju controller on it.
+
+1. [Microk8s](https://juju.is/docs/olm/microk8s)
+2. [Google Cloud (GKE)](https://juju.is/docs/olm/google-kubernetes-engine-(gke))
+3. [Amazon Web Services (EKS)](https://juju.is/docs/olm/amazon-elastic-kubernetes-service-(amazon-eks)#heading--install-the-juju-client)
+4. [Microsoft Azure (AKS)](<https://juju.is/docs/olm/azure-kubernetes-service-(azure-aks)>)
+
+## 3. Deploy charmed Magma Orchestrator
+
+From your Ubuntu machine, create an `overlay.yaml` file that contains the following content:
+
+```yaml
+applications:
+  orc8r-certifier:
+    options:
+      domain: <your domain name>
+```
+
+Replace `<your domain name>` with your domain name.
+
+Deploy Orchestrator:
+
+```bash
+juju deploy magma-orc8r --overlay overlay.yaml --trust --channel=edge
+```
+
+The deployment is completed when all services are in the `Active-Idle` state.
+
+## 4. Import the admin operator HTTPS certificate
+
+Retrieve the self-signed certificate:
+
+```bash
+juju scp --container="magma-orc8r-certifier" orc8r-certifier/0:/var/opt/magma/certs/..data/admin_operator.pfx admin_operator.pfx
```

Comment:
I just want to make sure that the double `..` in front of `data` are not a typo. I have never seen a path as special as this before. :innocent:

Reply:
Yeah that's intended :) We will change this soon though but if you are to deploy it today this would be how you retrieve the admin_operator pfx package.